ansible/configs/ocp4-disconnected-osp-lab/files/admin_host_requirements.txt

```diff
@@ -0,0 +1,79 @@
+ansible==2.7.9
```

Comment:
2.8.6

Reply:
The admin host is currently running 2.7.9, so I tried to align with that. However, to deploy this config it will need to likely run in a venv anyhow so I'm perfectly happy to rev back up. Most of the development was done with 2.8.5 & 2.8.6.